pcdsdevices/mv_interface.py

```diff
@@ -145,6 +147,24 @@ def umvr(self, delta, timeout=None):
         """
         self.umv(delta + self.wm(), timeout=timeout)
 
+    def mv_ginput(self, timeout=None):
+        """
+        Moves to a location the user clicks on.
+        """
+        print("Select new motor x-position in current plot by mouseclick")
+        if not pylab.get_fignums():
+            upperLimit = 0
+            lowerLimit = self.limits[0]
+            if self.limits[1] == 0:
+                 upperLimit = 100
+            else:
+                 uppderLimit = self.limits[1]
```

Comment:
typo right here `uppderLimit` instead of `upperLimit`

Reply:
fixed